From 9221efef7968463f3e3d9ce79ea72eaca082e73f Mon Sep 17 00:00:00 2001
From: Anton Khirnov <anton@khirnov.net>
Date: Wed, 31 Oct 2012 16:54:16 +0100
Subject: [PATCH 169/278] lavf: fix av_interleaved_write_frame() doxy.

A sentence was mistakenly inserted in the middle of another sentence.
---
 libavformat/avformat.h |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/libavformat/avformat.h b/libavformat/avformat.h
index 0961308..3e10496 100644
--- a/libavformat/avformat.h
+++ b/libavformat/avformat.h
@@ -1463,10 +1463,10 @@ int av_write_frame(AVFormatContext *s, AVPacket *pkt);
  * @param s media file handle
  * @param pkt The packet containing the data to be written. Libavformat takes
  * ownership of the data and will free it when it sees fit using the packet's
- * This can be NULL (at any time, not just at the end), to flush the
- * interleaving queues.
  * @ref AVPacket.destruct "destruct" field. The caller must not access the data
  * after this function returns, as it may already be freed.
+ * This can be NULL (at any time, not just at the end), to flush the
+ * interleaving queues.
  * Packet's @ref AVPacket.stream_index "stream_index" field must be set to the
  * index of the corresponding stream in @ref AVFormatContext.streams
  * "s.streams".
-- 
1.7.9.5

